downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/DatabaseDumps.java

```diff
@@ -0,0 +1,79 @@
+package org.reactome.release.downloadDirectory;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.lang.ProcessBuilder.Redirect;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import java.util.zip.GZIPOutputStream;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.gk.persistence.MySQLAdaptor;
+
+public class DatabaseDumps {
+	private static final Logger logger = LogManager.getLogger();
+	
+	public static void execute(MySQLAdaptor dba, String releaseNumber, String username, String password, String host, int port, String testReactomeDatabase) throws IOException, InterruptedException {
+		// Take mysqldumps of 'stable_identifiers' and 'test_reactome_##' and compress them using gzip.
+		
+		//TODO: Refactor
+		
+		logger.info("Running DatabaseDumps step");
+		// Create databases folder that will hold the DB dumps
+		File databaseDir = new File(releaseNumber + "/databases");
+		if (!databaseDir.exists()) {
+			databaseDir.mkdir();
+		}
```

Comment:
You should be able to do a call to Files.createDirectories(releaseNumber + "/databases") here so you don't have to worry about creating a File object or checking if the directory exists.

Reply:
Yes, the actual logic required a Paths object, so for posterity: `Files.createDirectories(Paths.get(releaseNumber + "/databases"))`

Reply:
Ah okay.  I must've misread the usage!